app/frontend/wallet/account.ts

```diff
@@ -283,10 +283,14 @@ const Account = ({
     return txWitness
   }
 
-  async function getMaxSendableAmount(address: Address, sendAmount: SendAmount) {
+  async function getMaxSendableAmount(
+    address: Address,
+    sendAmount: SendAmount,
+    decimals: number = 0
```

Comment:
it seems a bit weird to me that wallet would need to be aware of the amount of decimals to compute the max sendable amount - I would expect it to deal with the "raw" integer amounts at all times and the frontend should be the one taking care of formatting the amount properly. I guess I'm missing some non-trivial assumption/dependency here

Reply:
It's returning `SendAmount` type, which includes both: `quantity` as "raw" number and formatted `fieldValue` which is string. I didn't change underlying logic, just added necessary information so we can still return the formatted value as before.

If I remember correctly, we needed to lift formatted value from local state to global store for some reason that I don't remember. I can try refactor it back to the local state and remove formatting from actions, but I assume this isn't very high in priority right now.